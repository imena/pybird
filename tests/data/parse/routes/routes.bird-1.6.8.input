0001 BIRD 1.6.8 ready.
1007-1.1.1.0/24     via 10.0.0.10 on v10 [peer1 2023-09-22] * (100) [AS10010i]
1008-   Type: BGP unicast univ
1012-   BGP.origin: IGP
        BGP.as_path: 10010
        BGP.next_hop: 10.0.0.10
        BGP.med: 1779
        BGP.local_pref: 100
        BGP.community: (0,12989) (0,13188)
1007-2.2.2.0/24     via 10.0.0.20 on v10 [peer20 2023-08-29] * (100) [AS10020e]
1008-   Type: BGP unicast univ
1012-   BGP.origin: EGP
        BGP.as_path: 10020
        BGP.next_hop: 10.0.0.20
        BGP.med: 0
        BGP.local_pref: 100
        BGP.community: (65535,64526) (65535,64520) (65535,11804)
1007-                   via 10.0.0.21 on v10 [peer21 2023-07-29] (100) [AS10020e]
1008-   Type: BGP unicast univ
1012-   BGP.origin: EGP
        BGP.as_path: 10020
        BGP.next_hop: 10.0.0.21
        BGP.med: 0
        BGP.local_pref: 100
        BGP.community: (65535,64526) (65535,64520)
1007-10.0.0.0/23      dev v10 [direct1 2023-03-16] * (240)
1008-   Type: device unicast univ
1007-3.3.3.0/24     via 10.0.0.10 on v10 [peer20 2023-08-29] * (100) [AS10010i]
1008-   Type: BGP unicast univ
1012-   BGP.origin: EGP
        BGP.as_path: 10010
        BGP.next_hop: 10.0.0.10
        BGP.med: 0
        BGP.local_pref: 100
        BGP.community: (rt,0,10010) (0,13188)
0000