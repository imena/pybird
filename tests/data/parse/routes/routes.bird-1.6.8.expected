[
    {
        "prefix": "1.1.1.0/24",
        "peer": "10.0.0.10",
        "interface": "v10",
        "source": "peer1",
        "time": "2023-09-22",
        "best": true,
        "type": "BGP",
        "scope": "unicast",
        "family": "univ",
        "BGP": {
            "origin": "IGP",
            "as_path": "10010",
            "next_hop": "10.0.0.10",
            "med": "1779",
            "local_pref": "100",
            "community": "0:12989 0:13188"
        }
    },
    {
        "prefix": "2.2.2.0/24",
        "peer": "10.0.0.20",
        "interface": "v10",
        "source": "peer20",
        "time": "2023-08-29",
        "best": true,
        "type": "BGP",
        "scope": "unicast",
        "family": "univ",
        "BGP": {
            "origin": "EGP",
            "as_path": "10020",
            "next_hop": "10.0.0.20",
            "med": "0",
            "local_pref": "100",
            "community": "65535:64526 65535:64520 65535:11804"
        }
    },
    {
        "prefix": "2.2.2.0/24",
        "peer": "10.0.0.21",
        "interface": "v10",
        "source": "peer21",
        "time": "2023-07-29",
        "best": false,
        "type": "BGP",
        "scope": "unicast",
        "family": "univ",
        "BGP": {
            "origin": "EGP",
            "as_path": "10020",
            "next_hop": "10.0.0.21",
            "med": "0",
            "local_pref": "100",
            "community": "65535:64526 65535:64520"
        }
    },
    {
        "prefix": "10.0.0.0/23",
        "peer": null,
        "interface": "v10",
        "source": "direct1",
        "time": "2023-03-16",
        "best": true,
        "type": "device",
        "scope": "unicast",
        "family": "univ"
    }
]